tpl-implementations/include/experimental/__p1673_bits/kokkos-kernels/blas3_gemm_kk.hpp

```diff
@@ -0,0 +1,173 @@
+
+#ifndef LINALG_TPLIMPLEMENTATIONS_INCLUDE_EXPERIMENTAL___P1673_BITS_KOKKOSKERNELS_GEMM_HPP_
+#define LINALG_TPLIMPLEMENTATIONS_INCLUDE_EXPERIMENTAL___P1673_BITS_KOKKOSKERNELS_GEMM_HPP_
+
+#include "signal_kokkos_impl_called.hpp"
+
+namespace KokkosKernelsSTD {
+
+namespace gemm_impl{
+template <class size_type>
+constexpr bool static_extent_match(size_type extent1, size_type extent2)
+{
+  return extent1 == std::experimental::dynamic_extent ||
+         extent2 == std::experimental::dynamic_extent ||
+         extent1 == extent2;
+}
+} //end gemm_impl namespace
+
+//
+// overwriting gemm: C = AB
+//
+// for now, specialize for default_accessor
+// https://github.com/kokkos/stdBLAS/issues/122
+//
+template<class ExeSpace,
+	 class ElementType_A,
+         std::experimental::extents<>::size_type numRows_A,
+         std::experimental::extents<>::size_type numCols_A,
+         class Layout_A,
+         class ElementType_B,
+         std::experimental::extents<>::size_type numRows_B,
+         std::experimental::extents<>::size_type numCols_B,
+         class Layout_B,
+         class ElementType_C,
+         std::experimental::extents<>::size_type numRows_C,
+         std::experimental::extents<>::size_type numCols_C,
+         class Layout_C>
+requires (Layout_A::template mapping<std::experimental::extents<numRows_A, numCols_A>>::is_always_unique() &&
```

Comment:
[required, please]

Would you consider using `MDSPAN_TEMPLATE_REQUIRES` instead of the `requires` keyword, so that the code still compiles with C++ < 20 ?  The macro is defined [here](https://github.com/kokkos/mdspan/blob/6c18e327f603329c596c09fa09e3519c0031aed2/include/experimental/__p0009_bits/macros.hpp#L176) and there are several examples of its use in the reference implementation.